crates/rpc/rpc/src/eth/pubsub.rs

```diff
@@ -244,6 +254,13 @@ where
     fn pending_transaction_stream(&self) -> impl Stream<Item = TxHash> {
         ReceiverStream::new(self.pool.pending_transactions_listener())
     }
+
+    /// Returns a stream that yields all transactions emitted by the txpool.
+    fn full_pending_transaction_stream(
+        &self,
+    ) -> impl Stream<Item = NewTransactionEvent<<Pool as TransactionPool>::Transaction>> {
+        ReceiverStream::new(self.pool.new_transactions_listener())
```

Comment:
this isn't quite accurate because this yields all transactions, we need to filter_map them by their subpool